app/models/admin_only/member_app_waiting_reason.rb

```diff
@@ -12,15 +12,25 @@ def other_reason_placeholder?
 
 
     # This is effectively a CONSTANT that is used in the UI
-    def self.other_reason_name
-      I18n.t('admin_only.member_app_waiting_reasons.other_custom_reason')
+    def self.other_reason_name(locale = I18n.locale)
+      I18n.t('admin_only.member_app_waiting_reasons.other_custom_reason', locale)
```

Comment:
The second parameter to the translate method should be an options hash. As it stands now the locale is ignored, and the method always returns the swedish translation

I18n.t('admin_only.member_app_waiting_reasons.other_custom_reason', locale: locale)